app/services/add_remote_tags.rb

```diff
@@ -1,6 +1,10 @@
 class AddRemoteTags < RemoteTaggingService
   def process(tag)
-    post_request(object_url, tag)
+    post_request(object_url, [tag])
```

Comment:
The class name `AddRemoteTags`, and the change here indicates a single tag. Should we change the class name or allow input to be an array?

Reply:
@bzwei we are adding one tag at a time when we do the link and also when we do the unlink. So its still singular for us might be plural for remote services